release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -993,21 +1092,190 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
-+
-This bug fix ensures Kubernetes is used as the source of truth. OVN-Kubernetes now starts correctly and keeps both OVN and Kubernetes in sync on startup. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1915295[*BZ#1915295*])
+* Bare metal installations on installer-provisioned infrastructure no longer silently skip writing an image when invalid root device hints are provided. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1886327[*BZ#1886327*])
+* Previously, incomplete boot mode information for Supermicro nodes caused deployment by using Redfish to fail. That boot mode information is now included. As a result, Supermicro nodes can be deployed using Redfish. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1888072[*BZ#1888072*])
+
+* The Ironic API service that is embedded in bare-metal installer-provisioned infrastructure now uses four workers instead of eight workers. As a result, RAM usage is reduced. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1894146[*BZ#1894146*])
+
+*Builds*
+
+* Previously, Dockerfile builds could not change permissions of the `/etc/pki/ca-trust` directory or create files inside it. This issue was caused by fixing link:https://bugzilla.redhat.com/show_bug.cgi?id=1826183[*BZ#1826183*] in version 4.6, which added support for HTTPS proxies with CAs for builds and *always* mounted `/etc/pki/ca-trust`, which prevented builds that included their own CAs or modified the system trust store from working correctly at runtime. The current release fixes this issue by reverting Bug 1826183. Now, builder images that include their own CAs work again. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1891759[*BZ#1891759*])
+
+* Previously, after upgrading from {product-title} version 4.5 to version 4.6, running `git clone` from a private repository failed because builds did not add proxy information to the Git configuration that was used to pull the source code. As a result, the source code could not be pulled if the cluster used a global proxy and the source was pulled from a private Git repository. Now, Git is configured correctly when the cluster uses a global proxy and the `git clone` command can pull source code from a private Git repository if the cluster uses a global proxy. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1896446[*BZ#1896446*])
+
+* Previously, the node pull secret feature did not work: node pull secrets were not used if `forcePull: true` was set in the Source and Docker strategy builds. As a result, builds failed to pull images that required the cluster-wide pull secret. Now, node pull secrets are always merged with user-provided pull secrets. As a result, builds can pull images when `forcePull: true` is set, and the source registry requires the cluster-wide pull secret. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1883803[*BZ#1883803*])
```

Comment:
`Previously, the node pull secret feature did not work:`
Probably end this with a `.` and start next sentence?